src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -156,13 +159,20 @@ private void writeToAccessLogAndEventType(final RequestLogInfo requestLogInfo, f
                     requestLogInfo.contentEncoding,
                     requestLogInfo.acceptEncoding,
                     requestLogInfo.contentLength);
-            nakadiKpiPublisher.publish(accessLogEventType, () -> new JSONObject()
-                    .put("method", requestLogInfo.method)
-                    .put("path", requestLogInfo.path)
-                    .put("query", requestLogInfo.query)
-                    .put("app", requestLogInfo.user)
-                    .put("app_hashed", nakadiKpiPublisher.hash(requestLogInfo.user))
-                    .put("status_code", response.getStatus())
-                    .put("response_time_ms", timing));
+        }
+        nakadiKpiPublisher.publish(accessLogEventType, () -> new JSONObject()
+                .put("method", requestLogInfo.method)
+                .put("path", requestLogInfo.path)
+                .put("query", requestLogInfo.query)
+                .put("app", requestLogInfo.user)
+                .put("app_hashed", nakadiKpiPublisher.hash(requestLogInfo.user))
+                .put("status_code", response.getStatus())
+                .put("response_time_ms", timing));
+    }
+
+    private boolean shouldLogRequest(final RequestLogInfo requestLogInfo, final HttpServletResponse response) {
+        return !"POST".equals(requestLogInfo.method) ||
+                !PATTERN_POST_PUBLISH.matcher(requestLogInfo.path).matches() ||
```

Comment:
I think it would be "cheaper" to check like
```java
requestLogInfo.path.startsWith("/event-types/") && (requestLogInfo.path.endsWithWith("/events") || requestLogInfo.path.endsWithWith("/events/"))
```
because matching regexp for each request is quite expensive.